ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.19.2
+    tag: 1.22.2
```

Comment:
Just wondering when we moved away from using `latest` for libraries as per our standards. Not you that introduced this change as I can see it was pinned down beforehand. But if we are moving away from that practice we should really update the standards. 

> NB. For libraries the version of go will be pinned by the app using the library so for the ci jobs we want to use the latest version of go instead. So for libraries only, the tag in the above files should be `latest` instead of a specific version.

https://github.com/ONSdigital/dp-standards/blob/main/DEPENDENCY_UPGRADING.md#build-and-test-version-docker-image

Reply:
I had forgotten about that. Noted, thanks. I'm not aware that we're moving away from that policy.